ArtGallery.Services.Infrastructure.Build/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using ADotNet.Clients;
```

Comment:
run code cleanup

Reply:
applied code cleanup but no change found.

Reply:
![image](https://user-images.githubusercontent.com/89320816/139942484-5df82c81-4ba2-4808-a997-956c48887d75.png)

Reply:
yes, I did the same